.github/workflows/permissions_advisor.yml

```
@@ -0,0 +1,27 @@
+# Sample workflow to run the action
+name: Permissions Advisor
+
+permissions:
+  actions: read
+
+on:
+  workflow_dispatch:
+    inputs:
+      name:
+        description: 'The name of the workflow file to analyze'
+        required: true
+        type: string
+      count:
+        description: 'How many last runs to analyze'
+        required: false
+        type: string
+        default: "10"
+
+jobs:
+  advisor:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: GitHubSecurityLab/actions-permissions/advisor@v1
```

```suggestion
    - uses: GitHubSecurityLab/actions-permissions/advisor@v1 # v1.0.2-beta6
```

Maybe if I add this, Zizmor will be quiet and Dependabot will update it?